scripts/startup/start_log_viewer.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+WINDOW="${1}"
+LOGFILE="${2}"
+echo "Running $(basename "${0}") at $(date), WINDOW=${WINDOW}, LOGFILE=${LOGFILE}"
+
+# Wait for bash to be ready (not necessary, but makes
+# the window look tidier when you attach later).
+sleep 1s
+
+tmux send-keys -t "${WINDOW}" "date" C-m
+tmux send-keys -t "${WINDOW}" "cd \"${PANLOG}\"" C-m
+tmux send-keys -t "${WINDOW}" "less --follow-name \"${LOGFILE}\"" C-m
```

Comment:
I'll probably later submit a patch to this to add `grc` in front of the `less` (with appropriate checks if it exists) as I think its nice to have colorized log files. However that requires having a proper `grc` config file since we don't show standard log messages.